pkg/k8s/init.go

```diff
@@ -64,7 +64,9 @@ func retrieveNodeInformation(ctx context.Context, nodeGetter k8sGetter, nodeName
 	mightAutoDetectDevices := option.MightAutoDetectDevices()
 	var n *nodeTypes.Node
 
-	if option.Config.IPAM == ipamOption.IPAMClusterPool || option.Config.IPAM == ipamOption.IPAMClusterPoolV2 {
+	if option.Config.IPAM == ipamOption.IPAMClusterPool ||
+		option.Config.IPAM == ipamOption.IPAMClusterPoolV2 ||
+		option.Config.IPAM == ipamOption.IPAMMultiPool {
```

Comment:
Why is this needed? MultiPool should neither require nor set a local alloc CIDR.

Reply:
@gandro thanks for the review and catching this leftover change. Now that this PR no longer uses the `Node` type to store pod IP pools, this change is unneeded. I will remove it.